contracts/rewards/interfaces/IAutoCompoundingRewards.sol

```diff
@@ -93,6 +98,20 @@ interface IAutoCompoundingRewards is IUpgradeable {
      */
     function enableProgram(Token pool, bool status) external;
 
+    /**
+     * @dev sets the number of programs to process the rewards for, when auto-trigerred
```

Comment:
should probably say "auto processed"

Reply:
There's a spelling error in `trigerred`, which is why it hasn't been replaced with 'auto-processed' earlier on;
We need to consider the general naming convention in this context anyway (`processRewards` vs `autoProcess`).

Reply:
fixed